### PR TITLE
build: print output from docker to GRPC stream

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: internal/build/docker.go [hunk 1]
```diff
@@ -375,6 +375,9 @@ func readDockerOutput(reader io.Reader) (*json.RawMessage, error) {
 			return nil, fmt.Errorf("decoding docker output: %v", err)
 		}
 
+		// TODO(Han): make me smarter! 🤓
+		logger.Get(ctx).Info(fmt.Sprintf("%+v\n", message))
```
Comment (nicks) [inline internal/build/docker.go]: wonder if we need Infof and friends

File: internal/build/docker_test.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ func TestDigestFromSingleStepOutput(t *testing.T) {
 `
 
 	expected := digest.Digest("sha256:11cd0b38bc3ceb958ffb2f9bd70be3fb317ce7d255c8a4c3f4af30e298aa1aab")
-	actual, err := getDigestFromBuildOutput(bytes.NewBuffer([]byte(input)))
+	actual, err := getDigestFromBuildOutput(context.Background(), bytes.NewBuffer([]byte(input)))
```
Comment (nicks) [inline internal/build/docker_test.go]: i think there might be a helper method to create contexts with loggers for testing? if not, can we add one?

File: internal/build/docker.go (hunk 1 above)
Comment (landism) [reply internal/build/docker.go]: whoops, maybe I just named it poorly
`func (l logger) Info(format string, a ...interface{})`

File: internal/build/docker_test.go (hunk 2 above)
Comment (jazzdan) [reply internal/build/docker_test.go]: Oooh thanks for pointing that out!

**Pull request merged**